ordered_product/models.py

```diff
@@ -44,6 +44,7 @@ def decrease_quantity(self, qn):
 
     def increase_quantity(self, qn):
         self.quantity += qn
+        self.save()
```

Comment:
Change this line to this
```suggestion
        self.save_ordered_product()
```